sdk-python/livescoreapi.py

```diff
@@ -7,17 +7,29 @@
 """
 
 import urllib.request
+import requests
 
 
-class livescores:
+class Livescores:
+
+    api_url = ''
+    api_key = ''
+    api_secret = ''
+
+    def __init__(self, api_url, api_key, api_secret):
+        self.api_url = api_url
+        self.api_key = api_key
+        self.api_secret = api_secret
     
-    def get_all(key,secret):
-        html_file = urllib.request.urlopen("http://livescore-api.com/api-client/scores/live.json?key=" + key + "&secret=" + secret)
-        json_data = html_file.read()
-        return json_data
+    def get_all_livescores(self):
+       livescores = requests.get('{}scores/live.json?key={}&secret={}'.format(self.api_url, self.api_key, self.api_secret))
+       return livescores.json()
     
+    def get_livescores_by_country(self, country_id):
+        country_livescores = requests.get('{}scores/live.json?key={}&secret={}&country={}'.format(self.api_url, self.api_key, self.api_secret, country_id))
+        return country_livescores.json()     
 
-        
-                
-    
 
+    def get_livescores_by_league(self, league_id, language_id):
+        league_livescores = requests.get('{}scores/live.json?key={}&secret={}&league={}&lang={}'.format(self.api_url, self.api_key, self.api_secret, league_id, language_id))
```

Comment:
We need to have a workaround if the user does not provide language, because it is optional and we will not add it to the URL in that case.